README.md

```diff
@@ -100,7 +100,7 @@ The IntelliJ IDE supports Maven out of the box and offers a plugin for Scala dev
 * IntelliJ download: [https://www.jetbrains.com/idea/](https://www.jetbrains.com/idea/)
 * IntelliJ Scala Plugin: [http://plugins.jetbrains.com/plugin/?id=1347](http://plugins.jetbrains.com/plugin/?id=1347)
 
-Check out our [Setting up IntelliJ](https://github.com/apache/flink/blob/master/docs/internals/ide_setup.md#intellij-idea) guide for details.
+Check out our [Setting up IntelliJ](docs/flinkDev/ide_setup.md#intellij-idea) guide for details.
```

Comment:
This should be an absolute path to `flink-apache.org`.

Reply:
Do you mean `https://ci.apache.org/projects/flink/flink-docs-master/flinkDev/ide_setup.html#intellij-idea`? This is the generated page of this doc, but not under `flink.apache.org`

Reply:
yes, i meant `https://ci.apache.org/projects/flink/flink-docs-master/flinkDev/ide_setup.html#intellij-idea`.

Reply:
then done. I think we should backport this to release-1.7? I don't know if to 1.6 or earlier(maybe no need)